aggregator/src/aggregator/collection_job_tests.rs

```diff
@@ -495,6 +495,65 @@ async fn collection_job_put_idempotence_time_interval() {
         .unwrap();
 }
 
+#[tokio::test]
+async fn collection_job_put_idempotence_time_interval_varied_collection_id() {
```

Comment:
I think a test for fixed size is justified as well, since that code path was modified as well.